crates/teloxide/src/update_listeners/polling.rs

```diff
@@ -333,15 +377,33 @@ impl<B: Requester> Stream for PollingStream<'_, B> {
     type Item = Result<Update, B::Err>;
 
     fn poll_next(mut self: Pin<&mut Self>, cx: &mut task::Context<'_>) -> Poll<Option<Self::Item>> {
+        log::trace!("polling polling stream");
```

Comment:
No, the first polling refers to the fact that we are in the `poll_next` method, while the second one is describing the struct ("polling stream", since it implement long polling of telegram)

Reply:
I'm not sure that matters though, that's just a `trace!` (so a level even less than `debug!`), ultimately it's only useful for debugging polling impl